tmk_core/common/keyboard.c

```diff
@@ -139,13 +139,48 @@ __attribute__ ((weak))
 void matrix_setup(void) {
 }
 
+/** \brief keyboard_pre_init_user
+ *
+ * FIXME: needs doc
```

Comment:
I assume you'll fix these before merging?

Reply:
Maybe?  Nothing else is actually documented.  And I'm not sure of the formatting for this still.